README.md

```diff
@@ -117,6 +117,21 @@ jobs:
       - uses: arduino/arduino-lint-action@v1
 ```
 
+A more complex workflow which passes two inputs to run the default checks on the projects in the repository but with permissive compliance and a complete output:
+
+```yaml
+on: [push, pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: arduino/arduino-lint-action@v1
+        with: 
+          verbose: true
+          compliance: permissive
```

Comment:
```suggestion
A more complex workflow which uses [inputs](#inputs) to configure the action for Library Manager ["update" mode](#library-manager) and strict compliance:

```yaml
on: [push, pull_request]
jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: arduino/arduino-lint-action@v1
        with:
          library-manager: update
          compliance: strict
```
Because Library Manager "update" mode will be the thing most often needed (once a library is in Library Manager it can't use the default setting any more), I think this is the ideal thing to demonstrate inputs. 

Strict compliance mode might end up being less popular, but maybe having it in the example will provide a little more encouragement for people to try for full best practices.